pvlib/pvsystem.py

```diff
@@ -811,6 +812,10 @@ def physicaliam(aoi, n=1.526, K=4., L=0.002):
     spa
     ashraeiam
     '''
+    zeroang = 1e-06
+
+    aoi = np.where(aoi == 0, zeroang, aoi)
```

Comment:
As I understood it, if the input angle is 0, tau will be nan. 
zeroang = 1e-06 is defined for all cases, because iam is defined as tau(theta) / tau(0°), so that tau(0°) has to be calculated anyway. I didn't actually add the line, but moved it up from further down.

I guess we could define tau0 as a constant in the function and handling aoi == 0 with np.where after the calculations are done.